### DISCUSSION TITLE
16-bit C++ compiler.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#112176] [Programming Help] [answered]
### Body

Hi,
 I am making an 8086 game, and I was wondering where I could get a 16-bit C++ compiler, so I can reduce the amount of Assembly code?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by VegaCaroMiguel** [comment]
Hi there @PyDOS8 
Borland Turbo C++ is a very nice one for DOS based-games:
1. It supports graphics mode selection, with very nice in-built functions in order to ramp up your in game ux experience.
2. Complex polygon modeling is incredible 
3. Dynamic Menu creation 
4. Animation support 
5. Overhead reduction 

CONS: 
Borland-dialect C++ may have differences even from the ISO-standardized versions of the language

**Reply by ghost** [reply]
Where can I get Borland Turbo C++?

**Reply by VegaCaroMiguel** [reply, marked as answer]
https://winworldpc.com/product/turbo-c/3x 
Have a nice one :)
